### PR TITLE
Add Pylint configuration to epicli devcontainer

### PR DESCRIPTION
**Pull request by seriva**
- Add Pylint configuration to epicli devcontainer
- Add ability to output junit format of pylint run
- Fix low hanging fruit pylint warning/errors:
    - Un-used imports 
    - Import order
    - Un-needed whitespace
    - Un-used variables
    - Operator fixes (==, is), (!=, is not)

### CODE REVIEW

File: core/src/epicli/cli/engine/DeleteEngine.py [hunk 1]
```diff
@@ -19,18 +15,18 @@ def __enter__(self):
...
 
         with TerraformRunner(cluster_model, docs) as tf_runner:
-            tf_runner.delete()     
-            
-        shutil.rmtree(self.build_directory, ignore_errors=True)     
+            tf_runner.delete()
+
+        shutil.rmtree(self.build_directory, ignore_errors=True)
 
         return 0
```
Comment (erzetpe) [inline core/src/epicli/cli/engine/DeleteEngine.py]: Add enter at the end.

**Review by erzetpe — CHANGES_REQUESTED** [review]

**Review by to-bar — APPROVED** [review]
:heart_eyes:

**Review by erzetpe — APPROVED** [review]

**Review by przemyslavic — APPROVED** [review]

**Pull request merged**